sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md

```diff
@@ -133,118 +59,33 @@ The authentication events trigger allows a function to be executed when an authe
-* Post the following payload using something like Postman or Fiddler.
-* Steps for using Postman can be found (Link TBD)
-
-```json
-{
-  "type":"microsoft.graph.authenticationEvent.TokenIssuanceStart",
-  "source":"/tenants/{tenantId}/applications/{resourceAppId}",
-  "data":{
-    "@odata.type": "microsoft.graph.onTokenIssuanceStartCalloutData",
-    "tenantId": "30000000-0000-0000-0000-000000000003",
-    "authenticationEventListenerId1": "10000000-0000-0000-0000-000000000001",
-    "customAuthenticationExtensionId": "10000000-0000-0000-0000-000000000002",
-    "authenticationContext1":{
-      "correlationId": "20000000-0000-0000-0000-000000000002",
-      "client": {
-        "ip": "127.0.0.1",
-        "locale": "en-us",
-        "market": "en-au"
-      },
-      "authenticationProtocol": "OAUTH2.0",
-      "clientServicePrincipal": {
-        "id": "40000000-0000-0000-0000-000000000001",
-        "appId": "40000000-0000-0000-0000-000000000002",
-        "appDisplayName": "Test client app",
-        "displayName": "Test client application"
-      },
-      "resourceServicePrincipal": {
-        "id": "40000000-0000-0000-0000-000000000003",
-        "appId": "40000000-0000-0000-0000-000000000004",
-        "appDisplayName": "Test resource app",
-        "displayName": "Test resource application"
-      },
-      "user": {
-        "companyName": "Nick Gomez",
-        "country": "USA",
-        "createdDateTime": "0001-01-01T00:00:00Z",
-        "displayName": "Dummy display name",
-        "givenName": "Example",
-        "id": "60000000-0000-0000-0000-000000000006",
-        "mail": "test@example.com",
-        "onPremisesSamAccountName": "testadmin",
-        "onPremisesSecurityIdentifier": "DummySID",
-        "onPremisesUserPrincipalName": "Dummy Name",
-        "preferredDataLocation": "DummyDataLocation",
-        "preferredLanguage": "DummyLanguage",
-        "surname": "Test",
-        "userPrincipalName": "testadmin@example.com",
-        "userType": "UserTypeCloudManaged"
-      }
-    }
-  }
-}
-```
-
-* You should see this response:
-
-```json
-{
-    "data": {
-        "@odata.type": "microsoft.graph.onTokenIssuanceStartResponseData",
-        "actions": [
-            {
-                "@odata.type": "microsoft.graph.provideClaimsForToken",
-                "claims": {
-                    "DateOfBirth": "01/01/2000",
-                    "CustomRoles": [
-                            "Writer",
-                            "Editor"
-                        ]
-                    }
-             }
-        ]
-    }
-}
-```
+* Open the project that was created in the prior step. (How to get started)
+* Follow [these](https://learn.microsoft.com/entra/identity-platform/custom-extension-tokenissuancestart-setup?tabs=visual-studio%2Cazure-portal&pivots=nuget-library#build-and-run-the-project-locally) steps to test your app locally.
 
 ## Troubleshooting
 
 * Visual Studio Code
```

Comment:
Are these general troubleshooting steps for Functions or specific to your extension?

Reply:
These are general troubleshooting steps for functions